cmd/thor/flags.go

```diff
@@ -142,4 +142,8 @@ var (
 		Value: 16,
 		Usage: "set tx limit per account in pool",
 	}
+	disableTelemetryFlag = cli.BoolFlag{
+		Name:  "disable-telemetry",
+		Usage: "disable telemetry server",
```

Comment:
Yeah, I thought about it too ! 
I looked at the pattern and it looks like the default for boolflags is false. And I was thinking that it would make sense for this feature to be active per default.

Doesn't seem to be possible to set a default value on BoolFlag ?

Reply:
Ah, turns out there is BoolTFlag, which defaults to true. Will implement that, just making sure what the default status of the metrics should be.

Reply:
Sorted 👍 
Metrics are disabled by default, making the (false-per-default) BoolFlag the best option.